frontend/packages/dev-console/src/utils/svg-utils.ts

```diff
@@ -31,6 +31,12 @@ function roundedHull1(polyPoints: Point[], hp: HullPaddingGetter): string {
   return `M ${p1} A ${padding},${padding},0,0,0,${p2} A ${padding},${padding},0,0,0,${p1}`;
 }
 
+// Return a point that extends a segment a given length
+export function extensionPoint(a: Point, b: Point, length: number): Point {
```

Comment:
This doesn't look to be used anywhere.

Reply:
yup, left over (but used in a dependent PR). I'll remove and add to that PR.